beacon-chain/core/electra/transition_no_verify_sig.go

```diff
@@ -93,6 +94,8 @@ func ProcessOperations(
 		return nil, errors.Wrap(err, "could not process deposit receipts")
 	}
 
-	// TODO: Process consolidations from execution header.
+	if err := ProcessConsolidationRequests(ctx, st, exe.ConsolidationRequests()); err != nil {
+		return nil, fmt.Errorf("could not process consolidation requests: %w", err)
```

Comment:
you could also do errors.Wrap like above but either is ok

Reply:
I think we should be moving away from errors.Wrap